library/src/main/java/com/cube/storm/ui/model/page/ListPage.java

```diff
@@ -19,12 +21,57 @@
  * @project LightningUi
  */
 @NoArgsConstructor @AllArgsConstructor
-@Accessors(chain = true) @Data @EqualsAndHashCode(callSuper=false)
+@Accessors(chain = true) @Data @EqualsAndHashCode(callSuper = false)
 public class ListPage extends Page
 {
 	public static String CLASS_NAME = "ListPage";
+	public static final String ENGLISH_LANGUAGE_CODE = "en";
```

Comment:
Might be able to get this from a Java standard lib, but also no issue as a standard like this shouldn't change

Reply:
Totally true, thanks Ben!! 😄